test/test_model.py

```diff
@@ -301,14 +300,12 @@ def test_relation_data_modify_remote(self):
         # this will fire more backend calls
         with self.harness._event_context('foo_event'):
             data_repr = repr(rel_db1.data)
-        # the CountEqual and weird (and brittle) splitting is to accommodate python 3.5
-        # TODO: switch to assertEqual when we drop 3.5
-        self.assertCountEqual(
-            data_repr[1:-1].split(', '),
-            ["<ops.model.Unit myapp/0>: {}",
-             "<ops.model.Application myapp>: <n/a>",
-             "<ops.model.Unit remoteapp1/0>: {'host': 'remoteapp1/0'}",
-             "<ops.model.Application remoteapp1>: {'secret': 'cafedeadbeef'}"])
+        self.assertEqual(
```

Comment:
Despite the comment, I'm not sure if this seems any less brittle (as a whitespace-sensitive multi-line string), but at least it's more legible.

Reply:
Yeah, I know! Six of one, half a dozen of the other... but I updated it anyway.